<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDayTasks</class>
 <widget class="MyDayTasks" name="MyDayTasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мой день</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MyDayTasks {
    background-color: #310E10; /* Black bean */
    color: #F4E3B2; /* Vanilla */
}

QLineEdit {
    background-color: #947268; /* Cinerious */
    color: #74070E; /* Blood red */
    border: 1px solid #74070E;
    border-radius: 4px;
    padding: 5px 10px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 2px solid #74070E;
}

QScrollArea {
    background-color: transparent;
    border: none;
}

QScrollBar:vertical {
    background: #947268; /* Cinerious */
    width: 10px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background: #74070E; /* Blood red */
    min-height: 20px;
    border-radius: 5px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    background: none;
}

QPushButton {
    background-color: #F4E3B2; /* Vanilla */
    color: #74070E; /* Blood red */
    border: none;
    border-radius: 4px;
    padding: 8px 15px;
    font-size: 14px;
    text-align: left;
}

QPushButton:hover {
    background-color: rgba(244, 227, 178, 0.8); /* Vanilla 80% */
}

QPushButton:pressed {
    background-color: rgba(244, 227, 178, 0.6); /* Vanilla 60% */
}

QCheckBox {
    color: #F4E3B2; /* Vanilla */
    spacing: 5px;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
}

QCheckBox::indicator:checked {
    background-color: #74070E; /* Blood red */
    border: 1px solid #74070E;
}

QCheckBox::indicator:unchecked {
    background-color: #947268; /* Cinerious */
    border: 1px solid #74070E;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="textInput">
         <property name="placeholderText">
          <string>Введите текст и нажмите Enter</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #947268;
color: #74070E;
border: 1px solid #74070E;
border-radius: 4px;
padding: 8px 12px;
font-size: 14px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: none;</string>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>776</width>
            <height>546</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #310E10;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="buttonsLayout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyDayTasks</class>
   <extends>QWidget</extends>
   <header>myDayTasks.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
